settings.py

```diff
@@ -6,7 +6,7 @@
 log = "Rsync_Dx.log"
 errorlog = "Rsync_Dx.errorlog"
 bgarray = "/mnt/bgarray/"
-
+alissa_vcf_upload = "/diaggen/software/production/alissa_vcf_upload"
```

Comment:
I would prefer a "/" at the end: "/diaggen/software/production/alissa_vcf_upload/"